packages/graph/clients/typescript/api.ts

```diff
@@ -477,6 +477,12 @@ export interface EntityType {
    * @memberof EntityType
    */
   title: string;
+  /**
+   *
+   * @type {string}
+   * @memberof EntityType
+   */
+  pluralTitle: string;
```

Comment:
Should be removed?

Reply:
I regenerated the OpenAPI specs and they got removed 🙂